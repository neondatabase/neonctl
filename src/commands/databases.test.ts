import { describe } from 'vitest';

import { test } from '../test_utils/fixtures';

describe('databases', () => {
  test('list', async ({ testCliCommand }) => {
    await testCliCommand([
      'databases',
      'list',
      '--project-id',
      'test',
      '--branch',
      'test_branch',
    ]);
  });

  test('create', async ({ testCliCommand }) => {
    await testCliCommand([
      'databases',
      'create',
      '--project-id',
      'test',
      '--branch',
      'test_branch',
      '--name',
      'test_db',
      '--owner-name',
      'test_owner',
    ]);
  });

  test('delete', async ({ testCliCommand }) => {
    await testCliCommand([
      'databases',
      'delete',
      'test_db',
      '--project-id',
      'test',
      '--branch',
      'test_branch',
    ]);
  });
});
