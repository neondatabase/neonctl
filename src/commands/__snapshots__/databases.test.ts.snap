// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`databases > create 1`] = `
"name: test_database
"
`;

exports[`databases > delete 1`] = `
"name: test_db
owner_name: test_owner
"
`;

exports[`databases > list 1`] = `
"- name: db1
  owner_name: user1
- name: db2
  owner_name: user2
- name: test_db
  owner_name: test_user
"
`;
